app/controllers/application_controller.rb

```diff
@@ -27,12 +25,4 @@ def current_merchant?
   def current_admin?
     current_user && current_user.admin?
   end
-
-  def current_merchant_employee?
-    @merchant && current_merchant? && current_user.merchant_id == @merchant.id
-  end
-
-  def current_merchant_employee_for_item?
-    @item.merchant && (current_merchant? && current_user.merchant_id == @item.merchant.id)
-  end
```

Comment:
I wanted to remove these because they're exposing extraneous data to views, authorization is handled in controllers and targets only relevant actions

Reply:
Great idea!!!